.github/workflows/cicd.yml

```diff
@@ -251,6 +251,25 @@ jobs:
       code_coverage: false
       configurations: '["NativeOnlyDebug", "NativeOnlyRelease"]'
 
+  # Run the regression driver tests on self-hosted runners (only for 2022).
+  regression_driver_ws2022:
+    # Always run this job.
+    # Only run this on repos that have self-host runners.
+    needs: regular
+    if: github.repository == 'microsoft/ebpf-for-windows' && (github.event_name == 'schedule' || github.event_name == 'pull_request' || github.event_name == 'push' || github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch')
+    uses: ./.github/workflows/reusable-test.yml
+    with:
+      pre_test: .\setup_ebpf_cicd_tests.ps1 -KmTracing $true -KmTraceType "file" -TestMode "Regression" -RegressionArtifactsVersion "0.15.0"
```

Comment:
When would we update this version? Does this get updated on every release? Or would we only update this when we have new breaking changes?

Reply:
Ideally the downlevel tests would be a matrix of supported versions.

Reply:
This is now https://github.com/microsoft/ebpf-for-windows/issues/3424.

Reply:
Tracked using https://github.com/microsoft/ebpf-for-windows/issues/3424